src/handlers/InputHandler.js

```diff
@@ -0,0 +1,25 @@
+import { InputView } from '../views/InputView.js';
+import Car from '../models/Car.js';
+import {
+  validateInputStrings,
+  validateGameAttempts,
+  validateParticipants,
+} from '../validations/InputValidation.js';
+
+export default class InputHandler {
+  static async getParticipants() {
+    const inputStrings = await InputView.readInput(
+      '참가자 이름을 입력해주세요 (이름은 쉼표(,) 기준으로 구분): ',
+    );
```

Comment:
```suggestion
    const input = await InputView.readInput(
      '참가자 이름을 입력해주세요 (이름은 쉼표(,) 기준으로 구분): ',
    );
```
콘솔 입력받는 쪽까지는 로직이 같은데 아래에 시도 횟수를 받을 때는 `input`을 사용하셨는데 여기서는 `inputString`을 사용하셔서 어색한 느낌이 있는 것 같은데 어떻게 생각하시나요???